### ISSUE TITLE
Fails to extract directories (for some zips)

### ISSUE THREAD

**Issue by mal** [maxogden/extract-zip#10]
Noticed here: https://github.com/mafintosh/electron-prebuilt/issues/57#issuecomment-140660022

Example zip: https://github.com/atom/electron/releases/download/v0.32.3/electron-v0.32.3-linux-x64.zip

This appears to stem from whether a directory is included in the zip's manifest(?).

**test/cats.zip**

```
[...]
/tmp/cat-extract-test/cats/
/tmp/cat-extract-test/cats/.DS_Store
/tmp/cat-extract-test/cats/deema3.5771930_std.JPG
/tmp/cat-extract-test/cats/Deema_019.204205705_std.jpg
/tmp/cat-extract-test/cats/empty/
/tmp/cat-extract-test/cats/gJqEYBs.jpg
/tmp/cat-extract-test/cats/orange/
[...]
```

vs

**electron-v0.32.3-linux-x64.zip**
```
[...]
/tmp/elec-extract-test/libgcrypt.so.11
/tmp/elec-extract-test/libnotify.so.4
/tmp/elec-extract-test/resources/atom.asar
[...]
```

Note the lack of a `/tmp/elec-extract-test/resources/`.

**Comment by ducky427** [comment]
I've also just come across this bug via the exact same sequence.

**Comment by maxogden** [comment]
Oops my bad, working on a fix now

**Comment by maxogden** [comment]
fixed in 1.1.1

**Comment by mal** [comment]
Great, thanks!
